quictrace/interface.go

```diff
@@ -0,0 +1,39 @@
+package quictrace
+
+import (
+	"time"
+
+	"github.com/lucas-clemente/quic-go/internal/ackhandler"
+	"github.com/lucas-clemente/quic-go/internal/protocol"
+	"github.com/lucas-clemente/quic-go/internal/wire"
+)
+
+// A Tracer traces a QUIC connection
+type Tracer interface {
+	Trace(protocol.ConnectionID, Event)
+	EmitAll() map[string][]byte
```

Comment:
What does this method do?

Reply:
It returns all traces. I'll rename it to `GetAllTraces`, and document it better.